pkg/cmd/apply.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/kubermatic/kubeone/pkg/credentials"
+	"github.com/kubermatic/kubeone/pkg/tasks"
+)
+
+type applyOpts struct {
+	globalOptions
+	NoInit      bool `longflag:"no-init"`
+	Force       bool `longflag:"force"`
+	AutoApprove bool `longflag:"auto-approve"`
+}
+
+func applyCmd(rootFlags *pflag.FlagSet) *cobra.Command {
+	opts := &applyOpts{}
+
+	cmd := &cobra.Command{
+		Hidden: true, // for now
```

Comment:
We should consider having this command as `kubeone alpha apply` instead of hiding it. The `alpha` approach is very common in the Kubernetes community and safer than just hiding the command.

Reply:
`kubeone alpha apply` is the next stage, when it will actually start working :D 
For now, while it's not doing what it's suppose to, and while we are moving it towards "working state" — it's hidden.